### DISCUSSION TITLE
I'd like to have a text icon, that says I can divide text into columns

### DISCUSSION THREAD

**Discussion by JoshuaBehrens** [shopwareArchive/meteor-icon-kit#32] [Request: Icon] [answered]
So basically view-compact but divided in half or in three pieces.

I want to do a pull request where you can set the columns of a text :)

![image](https://user-images.githubusercontent.com/1133593/212101065-4b28511f-72c5-42de-b22d-81a84f3793de.png)

Currently I use default-view-split and pause-s was an option as well, but it is not really easy to understand with these icons

**Comment by Weltraumakustik** [comment, marked as answer]
Thank you for your idea.
Your suggested icon will be part of the next (#35) release! 🎉

It will be available in both regular and solid. The smaller version, `line-column-xs`, will only be available as regular icon due to display issues of the solid pack.

<img width="93" alt="Bildschirm­foto 2023-01-20 um 17 14 07" src="https://user-images.githubusercontent.com/1750320/213748641-42a5d176-6605-4538-90fd-9c952e954e2d.png">
